xmrstak/backend/nvidia/nvcc_code/cuda_extra.cu

```diff
@@ -472,7 +480,7 @@ extern "C" int cuda_get_deviceinfo(nvid_ctx* ctx)
 		std::vector<int>::iterator it = std::find(arch.begin(), arch.end(), 20);
 		if(it == arch.end())
 		{
-			printf("WARNING: NVIDIA GPU %d: miner not compiled for CUDA architecture %d.\n", ctx->device_id, gpuArch);
+			printf("SKIP! (arch not included)\n");
```

Comment:
This removes important information, please revert this change.

Reply:
Duplicate information once it dumps all information in the detection "header"
Note there is no newline on the detect header so it would actually be on the same line for that matter.

Reply:
Point of patch being, we get all that info all the time, not just when something is caught.
So dumping the info twice is pointless?